src/mlpack/methods/dbscan/ordered_point_selection.hpp

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file ordered_point_selection.hpp
+ * @author Kim Sang-Yeon
+ *
+ * sequentially select the next point for DBSCAN.
```

Comment:
Pedantic style issue; this should start with a capital letter.

Reply:
Yes, I will do!!